examples/preview/Exporter SDK/ExportSTL/BinarySTL.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using vrcontext.walkinside.exporter;
+using System.IO;
+
+namespace ExportSimple
+{
+    class BinarySTL
+    {
+        // The scenenode reference of the Walkinside exporter SDK.
+        IntPtr m_SceneNode = IntPtr.Zero;
+        // Reference the root CAD hierarchy element in Walkinside model.
+        long m_RootBranch = -1;
+
+        /// <summary>
+        /// Initialize the walkinside exporter sdk. Making it ready to process STL geometry.
+        /// </summary>
+        /// <param name="projectpath">The folder where the walkinside model will be created.</param>
+        /// <param name="projectname">The name of the folder that contains the main walkinside project file (.vrp). </param>
+        /// <returns></returns>
+        public bool Initialize(string projectpath, string projectname)
+        {
+            // Initialize the SDK.
+            if (0 == Bindings.vrBegin())
+                return false;
+
+            // Define the Walkinside model location.
+            if (Bindings.vrSettingsFileNameW(projectpath + "\\" + projectname) == 0)
+                return false;
+            
+            // Define the axis transformation to orientate STL geometry according Walkinside Axis.
+            // Walkinside:          STL:
+            //     X = Right            X = Right
+            //     Y = Up               Y = ?
+            //     Z = Depth            Z = ?
+            VRT_VECTOR3 xa = new VRT_VECTOR3(1,  0,  0);
+            VRT_VECTOR3 ya = new VRT_VECTOR3(0,  0, -1); // ToDo : Possible STL axis is not well defined, and these values need to change depending model.
+            VRT_VECTOR3 za = new VRT_VECTOR3(0,  1,  0);
```

Comment:
Not a big fan of using vars in example code to learn API. Visual studio intelisense can get messed up with var types. And when learning new API intellisense is your friend :)